spec/activefedora/ordered_spec.rb

```diff
@@ -231,6 +231,29 @@ class Image < ActiveFedora::Base
       expect(subject.ordered_members).to eq [member, member2, member2, member]
     end
   end
+  describe "insert_target_id_at" do
+    it "can add between items" do
```

Comment:
if we're avoiding "should" in descriptions, we should (heh) avoid other models too, like "can."